kopf/utilities/loaders.py

```diff
@@ -25,6 +26,7 @@ def preload(paths, modules):
     """
 
     for path in paths:
+        sys.path.insert(0, os.path.abspath(os.path.dirname(path)))
```

Comment:
This might add duplicate paths to `sys.path`, if `paths` contains the paths to multiple files in the same directory. If no de-duplication is performed during importing this could slow down imports a bit.

Reply:
Indeed. 

But only in the case when many scripts from the same dir are put on the command-line:

```
kopf run di/script1.py ... dir/script100.py
```

Which is hard to imagine. Even 2 is rarely needed (almost never) — as we learned while gaining some experience using Kopf.

So, let it be this way for now.